paasta_tools/cli/cmds/logs.py

```diff
@@ -81,7 +84,7 @@ def add_subparser(subparsers):
         help=cluster_help,
     ).completer = completer_clusters
     status_parser.add_argument(
-        '-f', '-F', '--tail', dest='tail', action='store_true', default=True,
+        '-f', '-F', '--tail', dest='tail', action='store_true', default=False,
```

Comment:
kwa seemed to think so, also, just with this change it will default to tailing since that's currently the only thing ScribeLogReader is set to support